examples/user_guide/Pandas_API.ipynb

```diff
@@ -699,6 +706,24 @@
     "scatter_matrix(df, alpha=0.2, diagonal='kde')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since scatter matrix plots may generate a large number of points (the one above renders 120,000 points!), you man want to take advantage of the power provided by [Datashader](https://datashader.org/) to rasterize the off-diagonal plots:"
```

Comment:
```suggestion
    "Since scatter matrix plots may generate a large number of points (the one above renders 120,000 points!), you may want to take advantage of the power provided by [Datashader](https://datashader.org/) to rasterize the off-diagonal plots into a fixed-resolution representation:"
```